README.md

```diff
@@ -1,16 +1,87 @@
-# GridDB Foreign Data Wrapper for PostgreSQL
+GridDB Foreign Data Wrapper for PostgreSQL
+==========================================
 
-This PostgreSQL extension is a Foreign Data Wrapper (FDW) for [GridDB][1].  
-This version of griddb_fdw can work for PostgreSQL 11, 12, 13, 14 and 15. It is confirmed in GridDB 5.0.0.
```

Comment:
I think the information of supported version is important, so could you keep this information?

Reply:
Added again, @hrkuma.

Reply:
Thank you for support.